src/main/java/edu/usf/cutr/tba/model/TravelBehaviorRecord.java

```diff
@@ -81,6 +97,9 @@
     private Boolean mIsTalkBackEnabled;
     private Boolean mIsPowerSaveModeEnabled;
 
+    public List<TravelBehaviorInfo.LocationInfo> locationInfoListStart = null;
```

Comment:
Just to match the headers, I would change the references from `Start` to `Origin` and `End` to `Destination` just to be consistent, here and below.

Reply:
The change will be reflected in the next PR.